### DISCUSSION TITLE
[Request New Feature] Add Wifi status information to the GUI

### DISCUSSION THREAD

**Discussion by RaW79** [revk/xESP32-Faikout#567] [Ideas]
Hello,
kindly ask for implementation of standard Wifi information to the GUI like often shown in other ESP solutions like: 

AP1 Information: SSID XXX
                            RSSI XX%, -xx dBm
                            Mode 11n
                            Channel X
                            BSSID XXXXXXXX

Would make it much more comfortable to analyse Wifi situation for the Faikin.

Big Thx!

**Comment by revk** [comment]
I'll add some more info in next beta

**Reply by RaW79** [reply]
Thank you!
